README.md

```diff
@@ -99,48 +55,7 @@ bazel build //beacon-chain:beacon-chain
 bazel build //validator:validator
 ```
 
-## Step 1: Deploy a Validator Registation Contract
-
-Deploy the Validator Registration Contract into the chain of the running geth node by following the instructions [here](https://github.com/prysmaticlabs/prysm/blob/master/contracts/validator-registration-contract/deployVRC/README.md).
-
-## Step 2a: Running a Beacon Node as a Validator or Observer
-
-Make sure a geth node is running as a separate process according to the instructions from the previous section. Then, you can run a full beacon node as follows:
-
-```
-bazel run //beacon-chain --\
-  --web3provider  ws://127.0.0.1:8546 \
-  --datadir /path/to/your/datadir \
-  --rpc-port 4000 \
-  --validator
-```
-
-This will spin up a full beacon node that connects to your running geth node, opens up an RPC connection for sharding validators to connect to it, and begins listening for p2p events.
-
-To try out the beacon node in development by simulating incoming blocks, run the same command above but enable the `--simulator` and a debug level, log verbosity with `--verbosity debug` to see everything happening underneath the hood.
-
-```
-bazel run //beacon-chain --\
-  --web3provider  ws://127.0.0.1:8546 \
-  --datadir /path/to/your/datadir \
-  --rpc-port 4000 \
-  --validator \
-  --simulator \
-  --verbosity debug
-```
-
-Now, deposit ETH to become a validator in the contract using instructions [here](https://github.com/prysmaticlabs/prysm/blob/master/docs/VALIDATOR_REGISTRATION.md)
-
-If you don't want to deposit ETH and become a validator, one option is to run a beacon node as an Observer. A beacon observer node has full privilege to listen in beacon chain and shard chains activities, but it will not participate
-in validator duties such as proposing or attesting blocks. In addition, an observer node doesn't need to deposit 32ETH. To run an observer node, you discard the `--validator` flag.
-
-```
-bazel run //beacon-chain --\
-  --datadir /path/to/your/datadir \
-  --rpc-port 4000 \
-```
-
-### Running via Docker
+## Running via Docker
 
 To run the beacon node within a docker container, use the `//beacon-chain:image` target.
```

Comment:
We actually provide two docker images for public consumption:

gcr.io/prysmaticlabs/beacon-chain:latest
gcr.io/prysmaticlabs/validator:latest

Should we update these instructions to run without bazel, using docker only?

Reply:
Note: I haven't pushed these up yet, but I will do it when we tag the 0.0.0 release.

Reply:
Sure thing